glean-core/preview/src/lib.rs

```diff
@@ -108,6 +109,7 @@ pub fn initialize(cfg: Configuration, client_info: ClientInfoMetrics) -> Result<
         upload_enabled: cfg.upload_enabled,
         data_path: cfg.data_path.clone(),
         application_id: cfg.application_id.clone(),
+        platform: cfg.platform.clone(),
```

Comment:
Should preview have a default value like the other bindings? Something like `Rust on <OS>`.

Reply:
Yes! And you can use [this](https://searchfox.org/glean/rev/a259debf8250d74732d1442ffb0ddd3068234a9b/glean-core/src/lib.rs#683) to get the OS part... actually, can this be used for all the bindings, so that everything can be within glean-core? Maybe we don't need the FFI changes at all.

Reply:
>  Maybe we don't need the FFI changes at all.

Oh yes, that would be so much better!

Reply:
Only problem is if we use this we don't get tha language name only the OS. Instead of `Python on Windows` we would get `Windows`. I think we need the ffi part, we can definitely remove the OS name responsability from the bindings, but would still need the language binding name. If we are fine with not having the language name, then ok.

Reply:
Oh, you're correct. To make things consistent, let's just allow to set the "language bindings name" from the FFI configuration. Let's take the OS from Rust.